go/vt/automation/resharding_task.go

```diff
@@ -0,0 +1,123 @@
+// Copyright 2015, Google Inc. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package automation
+
+import (
+	"fmt"
+	"strings"
+
+	pb "github.com/youtube/vitess/go/vt/proto/automation"
+)
+
+// ReshardingTask is a cluster operation which allows to increase the number of shards.
+type ReshardingTask struct {
+}
+
+func splitShardListIntoKeyspaceAndShards(shardsWithKeyspace []string) (string, []string, error) {
```

Comment:
DONE

Thanks for being persistent here! I've found topo.ParseKeyspaceShardString which parses on "keyspace/shard" string.

I've changed the code to use it.